Directory.Build.props

```diff
@@ -25,7 +25,12 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
+    <NetFrameworkMinimum>net462</NetFrameworkMinimum>
```

Comment:
This should already be defined by Arcade

Reply:
Removed.